packages/admin-ui/ui/src/domain/orders/details/order-line/index.tsx

```diff
@@ -84,6 +84,10 @@ const ReservationIndicator = ({
   reservations?: ReservationItemDTO[]
   lineItem: LineItem
 }) => {
+  const { variant, isLoading } = useAdminVariantsInventory(
+    lineItem?.variant_id ?? ""
+  )
```

Comment:
suggestion:
```suggestion
  const { variant, isLoading } = useAdminVariantsInventory(
    lineItem?.variant_id,
    { enabled: !lineItem?.variant_id }
  )
```

Reply:
This might actually affect the `isLoading` property, so we might need to use `isFetching` instead.

Reply:
Also, toally non-blocking :)